### PR TITLE
Home page exceeds page's dimensions on Chromium-based browsers

### LINKED ISSUE #462: Home page exceeds page's dimensions on Chromium-based browsers

### PR DESCRIPTION
**Pull request by 0xjei**
## Description

To solve this, I set the `overflowX` property in the body to `hidden` (to prevent the content from overflowing horizontally into the parent element's box) and added `minWidth` and `maxWidth` instead of `width`, stretching the image to the maximum for the page layout size.

## Related Issue
#462

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Review by vplasencia — APPROVED** [review]

File: apps/website/src/app/page.tsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ export default function Home() {
                         alt="Midnight whispers image"
                         src="https://semaphore.cedoor.dev/midnight-whispers.jpg"
                         objectFit="cover"
-                        w="full"
+                        minWidth="100%"
```
Comment (cedoor) [inline apps/website/src/app/page.tsx]: We could not need this. I've tried it with `overflowX: "hidden"` only and it works. Could you double-check?

Comment (0xjei) [reply apps/website/src/app/page.tsx]: You mean, the `w="full"`?

Comment (cedoor) [reply apps/website/src/app/page.tsx]: Sorry, * we could not need this.

`overflowX: "hidden"` could be enough for making it work.

Comment (0xjei) [reply apps/website/src/app/page.tsx]: No, if you remove the parameter `minWidth="100%"` the images do not extend horizontally and you get something like this

image

Comment (cedoor) [reply apps/website/src/app/page.tsx]: Umh I'm trying it with `w="full"` and it works

Comment (0xjei) [reply apps/website/src/app/page.tsx]: Yes, maintaining `w="full"` works since is equivalent to min & max at 100%. Lmk which ones do you prefer

Comment (cedoor) [reply apps/website/src/app/page.tsx]: Mmh I'd reduce the number of changes in this PR to the bare minimum.

Comment (0xjei) [reply apps/website/src/app/page.tsx]: Better for consistency across images, see last commit

Comment (cedoor) [reply apps/website/src/app/page.tsx]: There's still another image to update

Comment (cedoor) [reply apps/website/src/app/page.tsx]: @0xjei

**Review by cedoor — APPROVED** [review]

**Pull request merged**